modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -1636,16 +1636,16 @@ def apply_full_axis_select_indices(
             self._partition_mgr_cls.apply_func_to_select_indices_along_full_axis(
                 axis,
                 self._partitions,
-                func,
+                self._build_mapreduce_func(axis, func),
```

Comment:
this wrapper properly handles Series results in case of reduction along index axis, we apply this wrapper everywhere at the `dataframe.py`, even in non-reduction methods: [[1]](https://github.com/modin-project/modin/blob/6550939753c76e896ef2bfd65bb9468d6ad161d7/modin/core/dataframe/pandas/dataframe/dataframe.py#L275-L276), [[2]](https://github.com/modin-project/modin/blob/6550939753c76e896ef2bfd65bb9468d6ad161d7/modin/core/dataframe/pandas/dataframe/dataframe.py#L1372), [[3]](https://github.com/modin-project/modin/blob/6550939753c76e896ef2bfd65bb9468d6ad161d7/modin/core/dataframe/pandas/dataframe/dataframe.py#L1398-L1402), [[4]](https://github.com/modin-project/modin/blob/6550939753c76e896ef2bfd65bb9468d6ad161d7/modin/core/dataframe/pandas/dataframe/dataframe.py#L2001), it seems like a bug that the wrapper was missed here